src/components/taskDetails/index.tsx

```diff
@@ -158,6 +158,14 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
         }
     }
 
+    function getTaskEstimates(value: number | undefined) {
```

Comment:
Also `value` does not provide any valuable information. Please use a better name.

Reply:
Fixed this